types/plugin-error/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false
```

Comment:
Don't disable any linting rules.

Reply:
Disable is fine as there's an issue with this rule: https://github.com/Microsoft/dtslint/issues/88.